drivers/SmartThings/zigbee-button/profiles/eight-buttons-battery.yml

```diff
@@ -0,0 +1,38 @@
+name: eight-buttons-battery
```

Comment:
Hello, I have two discussion points for this profile.

A. Can you please change the names in the eight-buttons-battery profile ids from "endpoints" to "buttons". (Refer to four-buttons-battery for reference).This would help us ensure consistency with other profiles on this driver. 
B. I see that there are only four components, but this is an eight-button profile. Was this intentional?